generators/client/templates/angular/src/main/webapp/app/shared/language/_jhi-translate.directive.ts

```diff
@@ -7,7 +7,7 @@ import { Component, Input } from '@angular/core';
     selector: '[jhi-translate]',
     template: '<span [innerHTML]="jhiTranslate | translate:translateValues"></span>'
 })
-export class JhiTranslate {
+export class <%=jhiPrefixCapitalized%>Translate {
```

Comment:
This gives compile error at [shared-common.module.ts](https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts#L14). This needs to be changed to `JhiTranslate` in this case.